src/components/Section/Relationships/Relatives/Relatives.jsx

```diff
@@ -37,6 +37,31 @@ export class Relatives extends Subsection {
     this.storeKey = storeKey
   }
 
+  componentDidMount() {
+    if (this.props.List.branch) {
+      this.checkRelativesSubsectionErrors(this.props.List.branch.value)
+    }
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.List.branch.value !== this.props.List.branch.value) {
+      this.checkRelativesSubsectionErrors(this.props.List.branch.value)
+    }
+  }
+
+  checkRelativesSubsectionErrors = (value) => {
```

Comment:
What is the purpose of adding this? I would like to move away from this pattern of using `this.constructor.errors` on components, and we shouldn't need to explicitly "check" for errors if the form data changes since that is already done by the validators.